_styleguides/de.md

```diff
@@ -0,0 +1,557 @@
+# Stilregeln für die Übersetzung von Godot ins Deutsche
+
+Originalautor: Cerno_b (cerno.b@gmail.com)
+
+## Einleitung
+
+Dieses Dokument soll dabei helfen, die deutsche Übersetzung von Godot, sowie
+der Godot-Dokumentation, stilistisch zu vereinheitlichen. Dabei sollen
+Lesbarkeit, Klarheit und Benutzbarkeit besonders im Vordergrund stehen.
+
+Für die Übersetzung selbst ist das Tool
+[Weblate](https://hosted.weblate.org/projects/godot-engine/) im Einsatz, das in der
+[offiziellen Dokumentation](https://docs.godotengine.org/de/4.x/contributing/documentation/editor_and_docs_localization.html) näher
+erläutert wird. Die Bedienung von Weblate ist nicht Bestandteil dieses
+Dokuments.
+
+Weblate verfügt über ein Glossar, in dem häufig verwendete Begriffe gelistet
+werden, damit sie einheitlich übersetzt werden können. In diesem Dokument
+wird daher darauf verzichtet, die Übersetzung einzelner Begriffe zu
+klären. Sollte es bei der Übersetzung eines Begriffs Uneinigkeit geben,
+so sollte dies zunächst im [Chatraum für die deutsche
+Godot-Übersetzung](https://chat.godotengine.org/channel/translation-de) diskutiert und dann ins Glossar eingetragen werden.
+
+## Technische Hilfsmittel
+
+### Automatische Übersetzungen und Workflow
+
+Die Erfahrung zeigt, dass eine Übersetzung von längeren Texten, z.B. aus der
+Anleitung oder der Klassenreferenz oft zu schnelleren und besseren Ergebnissen
+führt, wenn man den Originaltext von einer KI vorübersetzen lässt.
+
+Ein sehr zu empfehlender Übersetzungsdienst ist hier die Website
+https://www.deepl.com, die hervorragende Erstübersetzungen zur Verfügung stellt
+und selbst Formatierungszeichen, wie Fettdruck oder Links in vielen Fällen
+korrekt in die Übersetzung übernimmt.
```

Comment:
We need to do due diligence on DeepL licensing before recommending it officially.

Reply:
I did this before starting to use it 

https://www.deepl.com/en/pro-license?tab=free

The important bits:

It's free:
> Both private and business use of the DeepL Translator (free) service are free of charge.

They don't claim copyright on translations
> DeepL SE does not obtain or assume any copyrights to the translations made by you using the translation service. 

Is this enough or do we need more?

Reply:
No that indeed seems reassuring enough :) Nice that you checked.